src/scripts/audit/tool.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+# Copyright 2020, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
We no longer use the phrase "All rights reserved" and some other corrections needed:

```
# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
```

Reply:
I want to know where you copied your line from because that place needs to be fixed too.

Reply:
Fixed. I don't remember exactly but I may have taken it from some old script which no longer exist in github.